packages/salesforcedx-vscode-soql/src/messages/i18n.ja.ts

```diff
@@ -22,7 +22,7 @@ export const messages = {
     'INFO: No default org found. Set a default org to use SOQL Builder. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org" to set one.',
   info_syntax_unsupported:
     'INFO: This syntax is not yet supported in SOQL Builder. Instead, use a text editor.',
-  info_file_save_success: 'We saved the results in the %s folder as: %s',
+  info_file_save_success: 'We saved the results as: %s',
```

Comment:
Already discussed this one with @ekapner on slack.

Reply:
Do we use language like "we?" in this sort of context?

Reply:
yeah apparently the SF messaging is moving to be more conversational